conf/messages.cy

```diff
@@ -962,6 +959,8 @@ global.error.pageNotFound404.title=Heb ddod o hyd i''r dudalen – 404
 site.link.giveDetails=Rhowch fanylion
 page.iht.application.debts.mortgage.inputText.value=Swm y morgais
 page.iht.questionnaire.easy-to-use.very-difficult=Anodd iawn
+iht.registration.timeout.p1=Ni wnaethom gadw'ch atebion.
+iht.application.timeout.p1=Gwnaethom gadw'ch cynnydd ar yr adroddiad ynghylch yr ystâd.
```

Comment:
Need to use double single quotes (escape) to render correctly in Welsh - currently displays without any quote